Dockerfile

```
@@ -0,0 +1,27 @@
+FROM ubuntu:latest
```

```suggestion
FROM ubuntu:24.04
```

Always best to select a version, and "manually" update to check regressions, than not clearly knowing what's being used under.